### ISSUE TITLE
Create project if it doesn't already exist

### ISSUE THREAD

**Issue by ConorSheehan1** [fescobar/allure-docker-service#135]
Would it be possible to add a param to `/send-results` or add a new endpoint, so that if results are sent to a project that doesn't exist, the project is created?

e.g.
```bash
curl -X POST "$ALLURE_SERVER/allure-docker-service/send-results?project_id=$PROJECT_ID&create_project=1" -H 'Content-Type: multipart/form-data' $FILES -ik
# create project $PROJECT_ID if it doesn't already exist
# send results to $PROJECT_ID
```

**Comment by fescobar** [comment]
@ConorSheehan1 it could be a new feature. I've added to the backlog.

**Comment by fescobar** [comment]
Hi @ConorSheehan1

**Comment by fescobar** [comment]
Hi @ConorSheehan1 
Your feature was released in version `2.13.6`
Use query parameter `force_project_creation=true` in `POST /send-results`
Here the documentation:
https://github.com/fescobar/allure-docker-service#force-project-creation-option
Thank you so much.

**Comment by ConorSheehan1** [comment]
Wow that was fast! Thank _you_ so much
